src/cli.rs

```diff
@@ -84,6 +84,29 @@ pub fn build_cli() -> App<'static, 'static> {
                 SubCommand::with_name("list")
                     .visible_alias("ls")
                     .about("List CloudTruth templates"),
+                SubCommand::with_name("getit")
+                    .about("Get implicit template")
+                    .arg(Arg::with_name("KEY")
+                        .required(true)
+                        .index(1))
+                    .arg(Arg::with_name("secrets")
+                        .long("secrets")
+                        .help("Display the values of secret parameters"))
+                    .arg(Arg::with_name("export")
```

Comment:
This starts getting into shell-specific functionality. Not all shells use `export`. Maybe it'd be better to pass the shell name and generate an appropriate file for that?

Reply:
That might be a nice addition to look directly at the shell. Right now it's up to the user to know and ask for export with the --export flag